Factory/PhpredisClientFactoryTest.php

```diff
@@ -45,13 +45,15 @@ public function testCreateFullConfig()
                 'connection_persistent' => true,
                 'prefix' => 'toto',
                 'serialization' => 'php',
+                'read_write_timeout' => 4,
             ),
             'alias_test'
         );
 
         $this->assertInstanceOf('\Snc\RedisBundle\Client\Phpredis\Client', $client);
         $this->assertSame('toto', $client->getOption(\Redis::OPT_PREFIX));
         $this->assertSame(1, $client->getOption(\Redis::OPT_SERIALIZER));
+        $this->assertSame(4., $client->getOption(\Redis::OPT_READ_TIMEOUT));
```

Comment:
Why the dot after 4?

Reply:
To say it's a float and not an int. Without the dot, the test failed because it returns a float.
This option is a float in the doc of phpredis.